dashboard/test/helpers/aichat_openai_responses_client_test.rb

```diff
@@ -17,7 +17,7 @@ class AichatOpenaiResponsesClientTest < AichatAiClientTest
     {
       'Accept'=>'*/*',
             'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
-            'Authorization'=>'Bearer',
+            'Authorization'=>/^Bearer(?: .*)?$/,
```

Comment:
Why is this change necessary?

Reply:
The old check only allowed `Bearer` with nothing following it. The new regex allows:
* `Bearer`
* `Bearer somekey`

This is needed because in order for developers to start rails console or server locally without AWS credentials, they are instructed to uncomment this line in locals.yml: https://github.com/code-dot-org/code-dot-org/blob/80b20c5edbbe336f63cbb5eb04c6fb14a3af8c1c/locals.yml.default#L10

After doing that, this test will fail because it now has an Authorization header of `Bearer localoverride` rather than just `Bearer`.